docs/README.md

```diff
@@ -80,6 +80,7 @@
 - [Sanic](./integrations/sanic.md)
 - [Chalice](./integrations/chalice.md)
 - [Starlette](./integrations/starlette.md)
+- [Starlite](./integrations/starlite.md)
```

Comment:
I think it might be worth renaming the page and link to litestar, since they changed name 😊

Reply:
Sure, can do...